docs/start-guide/how-to-run-simulators/scenario-simulator/run-on-cloud-env.md

```diff
@@ -0,0 +1,29 @@
+# Run on the cloud environment
+
+This page describes how to run the scenario simulator on the cloud environment.
```

Comment:
Everywhere else on the page says `cloud platform` rather than `cloud environment` so this should be consistent.

```suggestion
# Run on the cloud platform

This page describes how to run the scenario simulator on the cloud platform.
```